tests/lib/tools.py

```diff
@@ -411,8 +411,12 @@ def emulate_assertListEqual(self, seq1, seq2, msg=None, seq_type=None):
         diffMsg = '\n' + '\n'.join(
             difflib.ndiff(pprint.pformat(seq1).splitlines(),
                           pprint.pformat(seq2).splitlines()))
-        standardMsg = self._truncateMessage(standardMsg, diffMsg)
-        msg = self._formatMessage(msg, standardMsg)
+        # _truncateMessage and _formatMessage are not defined on Python 2.6; output the entire diff in that case
```

Comment:
This issue has always been here. The only reason I noticed it is because a test that uses assertListEqual failed for an unrelated issue this code raised an exception.